prow/pjutil/tot.go

```diff
@@ -27,8 +28,22 @@ import (
 	"k8s.io/test-infra/prow/config"
 	"k8s.io/test-infra/prow/kube"
 	"k8s.io/test-infra/prow/pod-utils/downwardapi"
+
+	"github.com/bwmarrin/snowflake"
+)
+
+var (
+	node *snowflake.Node
 )
 
+func init() {
+	var err error
+	node, err = snowflake.NewNode(1)
```

Comment:
We really shouldn't be using `1` as a seed for every process that needs to create ProwJobs. This value should be unique across all processes that might be creating ProwJobs in order to produce unique IDs if two processes try to create a ProwJob in the same millisecond.
Alternative seeds could be node id concatenated with the process id or a high precision timestamp might be sufficient. Anything besides a constant would be an improvement.